terraform/modules/hub/ingress.tf

```diff
@@ -290,14 +290,15 @@ resource "aws_route53_record" "ingress_www" {
 module "ingress_ecs_asg" {
   source = "./modules/ecs_asg"
 
-  ami_id              = data.aws_ami.ubuntu_bionic.id
-  deployment          = var.deployment
-  cluster             = "ingress"
-  vpc_id              = aws_vpc.hub.id
-  instance_subnets    = aws_subnet.internal.*.id
-  number_of_instances = var.number_of_apps * 2
-  domain              = local.root_domain
-  instance_type       = var.ingress_instance_type
+  ami_id           = data.aws_ami.ubuntu_bionic.id
+  deployment       = var.deployment
+  cluster          = "ingress"
+  vpc_id           = aws_vpc.hub.id
+  instance_subnets = aws_subnet.internal.*.id
+  min_size         = 2
+  max_size         = 20
```

Comment:
Not sure I understand why this is `20` and not `var.number_of_apps * 2`

Reply:
1. number_of_apps is the number of tasks for each of the non-{frontend,metadata,analytics} apps ... so number_of_apps is not related to anything in the ingress cluster
2. 20 is a somewhat arbitrary upper limit on the number of nodes in the cluster... we currently run about 5 nodes in that cluster in prod, so I just picked an upper limit 4x that ... the actual number of nodes will be determined by the resources required to hold number_of_frontend_app + number_of_metadata_apps + number_of_analytics_apps all with placement "spread"

Reply:
That makes sense. Thanks.